src/components/header/HeaderLock.tsx

```diff
@@ -0,0 +1,33 @@
+import React, { useMemo, useCallback } from 'react'
+import { palette } from 'styled-theme'
+import { useObservableState } from 'observable-hooks'
+import { useThemeContext } from '../../contexts/ThemeContext'
+import { HeaderLockWrapper } from './HeaderLock.style'
+import { ReactComponent as SettingsIcon } from '../../assets/svg/icon-lock.svg'
+import { MobileWrapper } from './Header.style'
+
+type Props = {
+  onPress?: () => void
+}
+
+const HeaderLock: React.FC<Props> = (_: Props): JSX.Element => {
```

Comment:
`_` is just a placeholder for an used variable and to avoid an eslint error (see https://github.com/thorchain/asgardex-electron/blob/master/.eslintrc#L39).` Better way something like this:

```
const HeaderLock: React.FC<Props> = (props: Props): JSX.Element => {
// Destructure props and provide default value for `onPress` - see next comment
const { onPress = () => {} } = props
...
}
```